# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../glibc/gcc_eh.patch.cross
# Copyright (C) 2012 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

A GCC cross compiler does not yield a libgcc_eh.so. The GCC and GlibC team
should smoke their good stuff together (and the GlibC freaks be fet with a
book about release management ...).

  - Rene Rebe <rene@exactcode.de>

--- glibc-2.16.0/Makeconfig.orig	2012-08-05 02:05:17.000000000 +0200
+++ glibc-2.16.0/Makeconfig.new	2012-08-05 02:11:01.949334945 +0200
@@ -523,7 +523,7 @@
 endif
 libgcc_eh := -Wl,--as-needed -lgcc_s $(libunwind) -Wl,--no-as-needed
 gnulib := -lgcc $(libgcc_eh)
-static-gnulib := -lgcc -lgcc_eh $(libunwind)
+static-gnulib := -lgcc  $(libunwind)
 libc.so-gnulib := -lgcc
 endif
 +preinit = $(addprefix $(csu-objpfx),crti.o)
